File: fsharp-giraffe-server/OpenAPI/src/model/JsonSuccessBaseAllOf.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic

module JsonSuccessBaseAllOf = 

  //#region JsonSuccessBaseAllOf

  //#region enums
  type ResultEnum = SuccessEnum of string  
  //#endregion

  type JsonSuccessBase_allOf = {
    Result : ResultEnum;
    Msg : string;
  }
  //#endregion
  

File: scala-gatling/src/gatling/scala/org/openapitools/client/model/JsonSuccessBaseAllOf.scala

package org.openapitools.client.model


case class JsonSuccessBaseAllOf (
    _result: String,
    _msg: String
)
object JsonSuccessBaseAllOf {
    def toStringBody(var_result: Object, var_msg: Object) =
        s"""
        | {
        | "result":$var_result,"msg":$var_msg
        | }
        """.stripMargin
}
